R/c-lib.R

```diff
@@ -77,16 +77,24 @@ use_rlang_c_library <- function() {
   }
 }
 
-download_rlang <- function() {
-  dest_zip <- fs::file_temp("rlang-src")
+download_package <- function(owner, repo) {
+  # Hardcoded for now
+  branch <- "main"
```

Comment:
In general, you can often skirt the `master` vs. `main` issue by using `HEAD` as the ref.